Library/Homebrew/diagnostic.rb

```diff
@@ -513,7 +514,7 @@ def check_user_path_3
           Homebrew's sbin was not found in your PATH but you have installed
           formulae that put executables in #{HOMEBREW_PREFIX}/sbin.
           Consider setting the PATH for example like so
-            echo 'export PATH="#{HOMEBREW_PREFIX}/sbin:$PATH"' >> #{shell_profile}
+            echo 'export PATH="#{HOMEBREW_PREFIX}/sbin:$PATH"' >> #{Utils::Shell.shell_profile}
```

Comment:
It would be still nice to fix this. Maybe by creating a new method like.
```ruby
<<-EOS.undent
 Consider setting the PATH for example like so
   #{Utils::Shell.prepend_path_in_shell_profile(HOMEBREW_PREFIX/"sbin")}
EOS
```

And
```ruby
def prepend_path_in_shell_profile(path)
  case preferred_shell
  when :bash, :zsh
    "echo '#{sh_quote(path)}:$PATH >> #{shell_profile}'
  when ...
  end
end